### PR TITLE
SC-14717  add opensearch logs

### PR DESCRIPTION
**Pull request by cpipilas**

### CODE REVIEW

File: docs/integration/opensearch-logs.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Once data is in, you can explore it using the built-in reports or create your own. For example, you can use the Errors report to see breakdown of what's wrong, for example, which nodes or components generate errors:
+
+<img
+  class="content-modal-image"
+  alt="OpenSearch Errors Report"
+  src="../../images/agents/opensearch-logs-errors.png"
+  title="OpenSearch Errors Report"
+/>
+
+Other built-in reports include:
```
Comment (otisg) [inline docs/integration/opensearch-logs.md:28]: Could we show them, @radu-gheorghe & @cpipilas ?

File: docs/integration/opensearch-logs.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+title: OpenSearch Logs Integration
+description: Sematext OpenSearch Logs integration allows you to check query distribution, analyze errors, deprecations and master logs and much more.
```
Comment (radu-gheorghe) [inline docs/integration/opensearch-logs.md:2]: master -> cluster manager

File: docs/integration/opensearch-logs.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  class="content-modal-image"
+  alt="OpenSearch Errors Report"
+  src="../../images/agents/opensearch-logs-errors.png"
+  title="OpenSearch Errors Report"
+/>
+
+Other built-in reports include:
+
+- **Queries**: breakdown of your queries
+- **Clustering**: logs produced by components that have to do with cluster coordination: master logs, logs related to a node joining/leaving a cluster and shard allocation
```
Comment (radu-gheorghe) [inline docs/integration/opensearch-logs.md:31]: master -> cluster coordinator

Comment (cpipilas) [reply docs/integration/opensearch-logs.md]: Fixed

File: docs/integration/opensearch-logs.md (hunk 2 above)
Comment (cpipilas) [reply docs/integration/opensearch-logs.md]: Fixed

**Review by radu-gheorghe — APPROVED** [review]
Sweet! Looks good to me!

**Pull request merged**